lib/get/get-full-source-space.js

```diff
@@ -94,5 +94,5 @@ function extractItems (response) {
 }
 
 function filterDrafts (items, includeDrafts) {
-  return includeDrafts ? items : items.filter((item) => !item.isPublished())
+  return includeDrafts ? items : items.filter((item) => !item.sys.publishedVersion)
```

Comment:
but then @axe312ger is right. You want to keep those items which have `sys.publishedVersion` set (otherwise you would be returning *only* the drafts)